run.sh

```diff
@@ -1 +1,7 @@
-python run.sh
\ No newline at end of file
+#!/usr/bin/env bash
+
+export WECHATY_PUPPET_SERVICE_NO_TLS_INSECURE_SERVER=true
+export WECHATY_PUPPET_SERVICE_ENDPOINT=127.0.0.1:8080
+export WECHATY_PUPPET_SERVICE_TOKEN=xiaoyan-1bfb-4090-8c88-bigbrother666
+
+python ~/AntigenWechatBot/run.py
```

Comment:
You should add the following code to activate the virtual environment.

```shell
conda activate wechaty
```

Second, just `python run.py` is alright to run. Because the the `run.sh` shell script exisit in the same directory.